src/server/pfs/server/driver.go

```diff
@@ -531,7 +532,8 @@ func (d *driver) makeCommit(pachClient *client.APIClient, ID string, parent *pfs
 		}
 
 		// BuildCommit case: Now that 'parent' is resolved, read the parent commit's
-		// tree (inside txn) and update the repo size
+		// tree (inside txn)
+		isOpen := false
```

Comment:
`isOpen` is a slightly confusing name to me, after reading a bit I think it indicates that the commit is being started by this call (which I had actually forgotten this call could do) but at first I thought it indicated that the commit was open before the call started. Maybe we could call it `isStartCommit` or something, I think that'd remove any confusion.